packages/zowe-explorer/README.md

```diff
@@ -23,26 +23,27 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.11.0](#whats-new-in-zowe-explorer-1.11.0)
+- [What's new in Zowe Explorer 1.12.0](#whats-new-in-zowe-explorer-1.12.0)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.11.0
+## What's new in Zowe Explorer 1.12.0
 
 New features:
 
-- Added login and logout functions for base profiles. You can now log in to API Mediation Layer and generate a token for your base profile.
+- Added the ability to edit data set attributes before allocation
+- Allowed filtering of member names from the Data Sets search bar
+- Reorganized the context menus and streamlined the visible icons
 
 Bug Fixes:
 
-- Fixed the empty profile folders in Favorites issue.
-- Fixed the initialization error that occurred when base profiles were used while being logged out from API ML.
-- Fixed the issue preventing the tree refresh function from updating extender profiles.
-- Fixed the issue causing jobs retrieval failure when using profiles with tokens.
+- Fixed the messaging displayed when handling inactive profiles and when updating profiles
+- Fixed the issue causing tree restructure when renaming a USS file or directory
+- Fixed the issue preventing issuing of commands when using profiles with tokens
```

Comment:
Can we change to "Fixed the error preventing issuing of commands"?

Sounds better than using _issue_ and _issuing_ in the same sentence. We could change the previous line to "error" too, just for continuity's sake.

Reply:
I'd suggest "Fixed the bug that prevented you from issuing commands when using the profiles with tokens"